pyiron_atomistics/atomistics/job/atomistic.py

```diff
@@ -741,9 +741,15 @@ def __getitem__(self, item):
         # new_structure.arrays['cells'] = new_structure.cell
         return new_structure
 
+    def _get_structure(self, frame, wrap_atoms=True):
```

Comment:
Nit: Codeacy complains that the signature is different from the base class implementation. Setting `frame-1` will get rid of this.

Reply:
Good catch!